src/igvfd/audit/file_set.py

```diff
@@ -8,6 +8,14 @@
 )
 
 
+def check_for_seq_config_files(value):
```

Comment:
can we rename the function to be `find_non_config_sequence_files(file_set)`

Reply:
yes, renamed